### PR TITLE
Support to send default service account jwt token for pipeline client.

### PR DESCRIPTION
**Pull request by hongye-sun**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/779)

### CODE REVIEW

File: sdk/python/build.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@
 # Setup:
 #   apt-get update -y
 #   apt-get install --no-install-recommends -y -q default-jdk
-#   wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.3.1/swagger-codegen-cli-2.3.1.jar -O /tmp/swagger-codegen-cli.jar
```
Comment (gaoning777) [inline sdk/python/build.sh]: Is the upgrade of swagger version necessary? I thought the 2.3.1 also works for this scenario.

**Comment by gaoning777** [comment]
Does this work for local jupyter notebook accessing a remote K8s cluster?

File: sdk/python/build.sh (hunk 1 above)
Comment (hongye-sun) [reply sdk/python/build.sh]: There is a problem when i ran the sdk in py 3.7 which is the only py env supported in cloud function. The generated uses async as parameter name which is a keyword in 3.7.

**Comment by hongye-sun** [comment]
This PR can only send default service account credential to IAP. It should work when the code running in GCE, gke, app engine and cloud function. I just tried it in GKE cluster and it is working fine.

**Pull request merged**